zap/src/main/java/org/parosproxy/paros/core/scanner/Alert.java

```diff
@@ -1081,12 +1082,48 @@ public Builder setAlertRef(String alertRef) {
             return this;
         }
 
-        /** @since 2.11.0 */
+        /** @since 2.11.0 @Deprecated 2.12.0 Replaced by {@link #setAlertTags(Map)} */
```

Comment:
I understood that the only methods that were going to be renamed were the history tag ones. `Alert#getTags` and `#setTags` do not have `Alert`.

Reply:
I may have lost track of where the conversation landed last week. That’s fine it simplifies the PR.